__tests__/app.test.js

```diff
@@ -73,4 +73,48 @@ describe("app", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    test("status:200, responds with the updated article", () => {
+      const updateVote = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(updateVote)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.articles).toEqual({
+            article_id: 3,
```

Comment:
wouldn't expect this back on a key called article**s** consider renaming to the singular